_posts/2020-10-16-changelog-133.markdown

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #133"
+date: 2020-10-19 12:00:00 +0300
+---
+
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5189 -->
+* Support configuring workspace features in `Cargo.toml`.
+  Now it is possible to enable or disable any specific workspace [feature](https://doc.rust-lang.org/cargo/reference/features.html) in the `Cargo.toml` file, and `Go to Declaration` and other code insight features will take them into account.
+
+  {% include gif-img.html path="/assets/posts/changelog-133/5189" w="700px" %}
+
+  **TODO: more info**
```

Comment:
Maybe just add `See more details in the [corresponding](https://github.com/intellij-rust/intellij-rust/pull/5189) issue`?